api/net/Areas/Editor/Controllers/LookupController.cs

```diff
@@ -136,7 +136,7 @@ public async Task<IActionResult> FindAllAsync()
         var metrics = _metricService.FindAll();
         var tagServices = _tagService.FindAll();
         var tonePools = _tonePoolService.FindAll();
-        var users = _userService.FindAll();
+        var users = _userService.FindByRoles(roles.Where(x => x != ClientRole.Subscriber.ToString().ToLower()));
```

Comment:
Please change to return users who have `ClientRole.Editor`

Reply:
Updated, thanks